envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/ssl/EnvoyHttpsDependencyTest.kt

```diff
@@ -21,22 +21,6 @@ class EnvoyCurrentVersionHttpsDependencyTest : EnvoyHttpsDependencyTest() {
     }
 }
 
-// TODO(https://github.com/allegro/envoy-control/issues/97) - remove when envoy < 1.14.0-dev will be not supported
```

Comment:
We no longer have any Envoy-s with this version on production.

Reply:
There is also a code that needs to be removed in EnvoyClustersFactory.kt

Reply:
You can also add "Fixes #97" to the description